openmetadata-ui/src/main/resources/ui/src/pages/TagPage/tag-page.less

```
@@ -0,0 +1,28 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+@import url('../../styles/variables.less');
+
+.tag-tabs {
+  .tag-overview-tab,
+  .assets-tab-container {
+    height: @tag-page-height;
+    overflow-y: scroll;
+  }
+}
+
+.tag-tabs {
+  .ant-tabs-nav {
+    margin: 0 !important;
+    padding: 0 16px;
+  }
+}
```

```suggestion
.tag-tabs {
  .tag-overview-tab,
  .assets-tab-container {
    height: @tag-page-height;
    overflow-y: scroll;
  }
  
   .ant-tabs-nav {
    margin: 0 !important;
    padding: 0 16px;
  }
}

```

Also why we need `!important` ?

---

`!important` is needed because the margin is not getting applied and it is causing a height scroll.